packages/ndla-ui/src/Navigation/NavigationBox.tsx

```diff
@@ -37,11 +37,11 @@ const StyledHeading = styled(Heading)`
 const StyledList = styled.ul`
   list-style: none;
   padding: 0;
-  display: flex;
-  flex-direction: column;
   gap: ${spacing.xxsmall};
 
   &[data-direction="horizontal"] {
+    display: flex;
+    flex-direction: column;
```

Comment:
Flytta disse ned for å unngå https://test.ndla.no/nb/subject:1:54b1727c-2d91-4512-901c-8434e13339b4